sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -679,8 +681,12 @@ def connect(self):
                 http_proxy_auth = (username, password)
         try:
             from websocket import create_connection
+            if self.parsed_custom_hostname!=None:
+                url = "wss://{}:{}/$servicebus/websocket/".format(self.parsed_custom_hostname,self.parsed_custom_port)
```

Comment:
This formatted URL is specific to the azure services, and so shouldn't be the pyamqp layer.
This should instead be formatted in the EventHubs/ServiceBus layer.